### PR TITLE
Part 6 views

### PR DESCRIPTION
**Pull request by henokkhm**
This pull request implements the UI according to the wireframes provided.

## Summary
- Add  Google font link
- Add CDN links of stylesheets
- Add custom stylesheet
- Add markup and styles for the following routes 
    - /users 
    - /users/:user_id 
    - /users/:user_id/posts 
    - /users/:user_id/posts/:post_id 
  
blog-app-screnshot

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,21 @@
-<h1>Details Page for a Specific User</h1>
+<section>
+  <%= render "components/user-card", user: @user  %>
+  <div class="bio-wrapper">
+    <p><%= @user.bio %></p>
+  </div>
+  <%if @user.posts.any? %>
```
Comment (Lucas-Masaba) [inline app/views/users/show.html.erb:6]: - [x] As mentioned in the project requirements, please use the last_three_posts` method you created in your model to render the last 3 posts.

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1 +1,31 @@
...
+    <ul class="posts-wrapper">
+      <% @user.posts.each do |post| %>
+        <li class="post-preview-card">
+          <a href="/users/<%= @user.id %>/posts/<%= post.id %>">
+            <%= render "components/post-preview-card", post: post %>
+
+            <%if post.comments.any? %>
+              <h4>Comments</h4>
+              <ul class="comments-wrapper">
+                <% post.comments[0, 5].each do |comment| %>
```
Comment (Lucas-Masaba) [inline app/views/posts/index.html.erb]: - [x] As mentioned in the project requirements, please use the last_five_comments` method you created in your Post model to render the last 3 posts.

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @henokkhm  👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights 🍾 
- Linter checks passing :heavy_check_mark: 
- Views are working as expected :gear: 
- Well-written commit messages :writing_hand: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @henokkhm, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
-  Implemented required changes. :heavy_check_mark: 
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct Gitflow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**